platform/linux-generic/pktio/loop.c

```diff
@@ -1,8 +1,6 @@
-/* Copyright (c) 2013-2018, Linaro Limited
- * Copyright (c) 2013-2023, Nokia Solutions and Networks
- * All rights reserved.
- *
- * SPDX-License-Identifier:     BSD-3-Clause
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright (c) 2013-2018 Linaro Limited
+ * Copyright (c) 2013-2023 Nokia Solutions and Networks
```

Comment:
I guess the company is still legally "Nokia Solutions and Networks", but should "Nokia" be used anyways? Or separate the two periods?

(There are a couple of places.)

Reply:
Agreed in a call that from now on only "Nokia" copyright will be used by us. No changes in this PR.